tests/image.test.js

```diff
@@ -254,6 +269,17 @@ describe('Image', () => {
     expect(dummyRoot.instance.image.mock.calls[0][0]).toBe(image.image.data);
   });
 
+  /* test('Should render a buffer gif image', async () => {
```

Comment:
This test currently makes the test runner hang, which is why I’ve commented it out.